jetty-core/jetty-quic/jetty-quic-api/src/main/java/org/eclipse/jetty/quic/api/frames/Frame.java

```diff
@@ -0,0 +1,59 @@
+//
+// ========================================================================
+// Copyright (c) 1995 Mort Bay Consulting Pty Ltd and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+// ========================================================================
+//
+
+package org.eclipse.jetty.quic.api.frames;
+
+public class Frame
+{
+    public static final int DEFAULT_MAX_SIZE = 16384;
+
+    private final long frameType;
+
+    public Frame(long frameType)
+    {
+        this.frameType = frameType;
+    }
+
+    public long getFrameType()
+    {
+        return frameType;
+    }
+
+    @Override
+    public String toString()
+    {
+        return "%s@%x".formatted(getClass().getSimpleName(), hashCode());
```

Comment:
The new "fashion" in `toString` implementations is to use the short name for the class, this is probably a good thing to do for such a deep package.   I'm wondering if we should add a method to `TypeUtil` like:

```java
    public static String toString(Object subject, String format, Object... args)
    {
        String combinedFormat = "%s@%x{" + format + "}";
        Object[] allArgs = new Object[args.length + 2];
        allArgs[0] = TypeUtil.toShortName(subject.getClass());
        allArgs[1] = subject.hashCode();
        System.arraycopy(args, 0, allArgs, 2, args.length);
        return combinedFormat.formatted(allArgs);
    }
```

I know this is kind of orthogonal to this PR, but you are introducing a lot of new classes that will be toStringed.

Reply:
Now using `TypeUtil`, but the larger change is for a different PR, if at all.